File: Library/models/Rating.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Library.models
{
    public class Rating
    {
        public int ID { get; set; }
        public int UserID { get; set; }
        public int TaskID { get; set; }
        public DateTime Date { get; set; }
        public int rating { get; set; }
        public string Message { get; set; }
    }
}

File: spa/src/types/rating.ts
export interface Rating {
    id: number;
    userID: number;
    taskID: number;
    date: Date;
    rating: number;
    message: string;
}

export interface RatingPlus {
    name: string;
    date: Date;
    rating: number;
    message: string;
}
